lib/plugins/inventory.js

```diff
@@ -463,6 +469,15 @@ function inject (bot, { version, hideErrors }) {
       })
       bot.emit(`confirmTransaction${click.id}`, false)
     }
+
+    function onRejectedError () {
```

Comment:
how come its called onRejectedError but you reply `accepted: true`, and you also comment out an emit, that's breaking for no reason, can you just write the packet in the if statement and revert the changes here?

Reply:
I have done that